### DISCUSSION TITLE
How to scan amazon-us from UK without using a VPN

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#2548] [Help]
If I try to scan amazon-us from the UK the website will geolocate my location to UK showing  "Deliver to United Kingdom" and as a result this won't show when stock is available and sold by amazon.com

I have tried changing the delivery address to US on the website when the bot is running but it just reverts back to the UK after a short time.

I have tried using a VPN with a US server however this is just far too slow at scanning when using that.

Anyone figured out a way around this?

Many thanks

**Comment by jiglesiasc** [comment]
Try using a US proxy on the config of street merchant. It works for me.

**Reply by lozbinks** [reply]
@jiglesiasc thanks for that, I had an feeling that may be a good option and did try it however I must have been doing something wrong as I just ended up with errors when I ran the bot.

Please could you let me know where the best place to get US proxies is? 

Also do they just go in the dotenv next to 

`PROXY_ADDRESS=
PROXY_PROTOCOL=
PROXY_PORT=`

and also how they should be formatted there?

Thanks again!

**Reply by jiglesiasc** [reply]
What I did, was to create a file named global.proxies on street merchant root directory. Inside that file you need to insert the proxy server IP address in this format: protocol://[user:password@]ip[:port]

Valid examples include: - socks5://1.2.3.4:3180 - socks5://abcd:efgh@1.2.3.4:5678 - http://1.2.3.4:80 - http://abcd:efgh@1.2.3.4:8080

For proxy server, you need a fast proxy server. I couldn't find one for free that works well so I went with privateproxy.me. They charge $5/month/IP

**Reply by lozbinks** [reply]
Top man! Much appreciated will give that a go shortly.
